spec/models/ingest/loader/process_csv_row_spec.rb

```diff
@@ -62,6 +62,18 @@
       end
     end
 
+    context 'with a string' do
+      it 'strips whitespace around values' do
+        data = {
+          'Customer Organisation' => '   Crown Commercial Service   '
```

Comment:
Should we have some sort of test around making sure we're not stripping values that aren't strings? Not entirely sure what we're protecting against, but we're making sure we're only doing it to Strings, so might be nice to have an example spec of what we're making sure we're not stripping.